### ISSUE TITLE
[Bug]

### ISSUE THREAD

**Issue by Fyoncle** [Stardust-Labs-MC/Incendium#81]
### What Minecraft version(s) does this apply to?

1.20

### Project version

5.3.5

### Mod or datapack version?

Mod (Fabric)

### Describe the bug

![image](https://github.com/Stardust-Labs-MC/Incendium/assets/63975256/f4cbc3ba-71c5-41b0-899e-84f80d418008)
Incompatible with yung's better nether fortresses, incendium structures breaks them.

### Additional context

I use 1.20.1

**Comment by catter1** [comment]
This isn't limited to YUNG's Better Nether Fortresses, this can happen with any structure. When deciding where to place a structure, Mojang gives us very limited configuration. This is what we have:
- Which biomes
- How rare
- An exclusion zone: a structure set where this structure cannot spawn next to

An exclusion zone should technically be the correct way to solve this. However, you can only define one exclusion zone per structure set. This is already used in Incendium to prevent the smaller structures from generating inside the larger structures. Because of this, we cannot also prevent other structures (such as YUNG's or even Vanilla) from intersecting, or vice versa. Not only that, but exclusion zones are implemented in a way, where the more you use, structures get more and more rare. They are generally not liked to be used due to it.

Tl;dr, there is nothing we can do about it unfortunately.
